File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Regions/Region.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Regions;

class Region  implements IRegion
{


    public IRegionDetailData $Details;
    public ITaxData $Tax;
    public IRegionList $Childs;
    public string $Description;
    public int $State;
    public int $OrderNo;
    public string $ID;
    public string $Name;
    public IRegion $Parent;
}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/Regions/Region.cs
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.Products;
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.RegionDetails;
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.Taxes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.RegionDetails;
using Ohtap.ECommerce.Domain.Models.Objects.Regions;
using Ohtap.ECommerce.Domain.Models.Objects.Taxes;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.Regions
{
    public class Region : RegionBasic, IRegion
    {
        public IRegionList Childs { get; set; }
        public ITaxData Tax { get; set; }
        IRegionDetailDataList IRegion.Details { get; set; }
    }
}
